packages/atomic/src/components/atomic-results-per-page/atomic-results-per-page.tsx

```diff
@@ -86,23 +88,25 @@ export class AtomicResultsPerPage implements InitializableComponent {
     }
   }
 
-  // TODO: use standard keyboard shortcuts KIT-949
   private buildChoice(choice: number) {
     const isSelected = this.resultPerPage.isSetTo(choice);
 
     return (
-      <Button
-        style="outline-neutral"
-        role="radio"
-        ariaChecked={`${isSelected}`}
-        ariaLabel={this.bindings.i18n.t('display-results-per-page', {
-          results: choice,
-        })}
-        onClick={() => this.resultPerPage.set(choice)}
-        class={`btn-page ${isSelected ? 'selected' : ''}`}
-        part={`button${isSelected && ' active-button'}`}
-        text={choice.toLocaleString(this.bindings.i18n.language)}
-      ></Button>
+      <input
```

Comment:
Should the `atomic-pager` use an `input` too to stay consistent?

Reply:
At first, I didn't think so since it has `aria-current`, but it looks like [aria-current is supported for `role="radio"`](https://www.w3.org/TR/wai-aria-1.1/#radio).

I'll create a new issue to make that change too.

Reply:
If so, I recommend extracting a shared functional component for this, similar to the `<Button>` you replaced here.

It can handle any accessibility concern, as well as UI (eg: ripple effect).